### ISSUE TITLE
Add compatibility for screen reader

### ISSUE THREAD

**Issue by chirag-jn** [IIIT-Delhi/sg-cw-iiitd#7]
Some of our users have to use screen readers for using the platform. The website should be able to incorporate that.
